flavors/documentation/action.yml

```diff
@@ -7,7 +7,7 @@ outputs:
     description: "0 if no source file has been updated, 1 if source files has been updated"
 runs:
   using: "docker"
-  image: "docker://oxsecurity/megalinter-documentation:v6.9.1"
+  image: "docker://oxsecurity/megalinter-documentation:v6"
```

Comment:
Is that the expected output for action files?

Reply:
GitHub actions looks for the branch/tag in the related repo, then it uses the related action.yml

When a new MegaLinter release is uploaded, it overrides "v6" image so v6 is always the latest v6 minor/patch version